templates/docker-compose.template

```diff
@@ -48,41 +52,32 @@ services:
     restart: always
     depends_on:
       - helium-miner
-    expose:
-      - "1681"
+    network_mode: host
+    extra_hosts:
+        - "helium-miner:127.0.0.1"
     labels:
       io.balena.features.sysfs: 1
       io.balena.features.procfs: 1
     volumes:
       - nebra:/var/nebra
 
   helium-miner:
-    image: nebraltd/hm-miner:{{ARCH}}-{{MINER_VERSION}}
-    depends_on:
-      - dbus-session
-      - diagnostics
-    expose:
-      - "1680"
-      - "4467"
-    ports:
-      - "44158:44158/tcp"
-    volumes:
-      - miner-storage:/var/data
-      - miner-log:/var/log/miner
-      - pktfwdr:/var/pktfwd
-      - dbus:/session/dbus
+    image: nebraltd/hm-gatewayrs:{{ARCH}}-{{GATEWAYRS_VERSION}}
+    network_mode: host
     cap_add:
       - SYS_RAWIO
-    devices:
-      - {{I2C_DEVICE}}:{{I2C_DEVICE}}
+    privileged: true
     restart: on-failure
     environment:
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
+      - SWARM_KEY_URI={{SWARM_KEY_URI}}
```

Comment:
as per comment above this will not work properly

Reply:
Yea, gonna open another PR just for that (against light-hotspot) to fix it as we have quite a bit active Syncrobit devices.